File: frontend/entities/MinutesAdded.ts
import Event from "./ActivityEvent";
import ParticipantEvent from "./ParticipantEvent";


export default class MinutesAdded {
    id: string;
    date: string;
    amount: number;
    participantEventId: string;
    participantEvent: ParticipantEvent | null;

    constructor(
        id: string,
        date: string,
        amount: number,
        participantEventId: string,
        participantEvent: ParticipantEvent | null,
    ) {
        this.id = id;
        this.date = date;
        this.amount = amount;
        this.participantEventId = participantEventId;
        this.participantEvent = participantEvent;
    }
}

File: backend/App.Domain/MinutesAdded.cs
using Base.Domain;

namespace App.Domain;

public class MinutesAdded : BaseEntityId
{
    public DateTime Date { get; set; }
    public int Amount { get; set; }
    public Guid ParticipantEventId { get; set; }
    public ParticipantEvent? ParticipantEvent { get; set; }
}